feat: add standalone rock usage documentation (#50)
canonical/opentelemetry-collector-rock: README.md

* do not use microk8s.kubectl (from snap) due to permission errors
    * use the kubectl snap instead
* sign the [CLA](https://ubuntu.com/legal/contributors)

## Usage

> If you want to run this OpenTelemtry Collector rock to observe a non-charmed deployment, all you need is to write a few yaml manifests.

Reviewer: ```suggestion
If you want to run this OpenTelemtry Collector rock to observe a non-charmed deployment, you will need to write a few yaml manifests.
```

Here are the manifests you need to deploy the Collector and a Node Exporter pod to get metrics from the local host:
- [`opentelemetry-collector.yaml`](https://github.com/canonical/opentelemetry-collector-rock/blob/c0b24df15627dc811472a1ce4589dcc85e380f40/tests/node_exporter/otel-collector.yaml)

> - [`node-exporter.yaml`](https://github.com/canonical/opentelemetry-collector-rock/blob/c0b24df15627dc811472a1ce4589dcc85e380f40/tests/node_exporter/node-exporter.yaml)

Author: Nope, fixed
